### PR TITLE
Refactor Dockerfiles to use new hekit command

### PR DESCRIPTION
**Pull request by denCalderon**
Using hekit.py on the docker build

### CODE REVIEW

**Review by hamishun — CHANGES_REQUESTED** [review]

File: kit/command_install.py [hunk 1]
```diff
@@ -36,4 +36,5 @@ def install_components(args):
         if component.skip():
             print(f"Skipping", comp_label)
```
Comment (hamishun) [inline kit/command_install.py:37]:
```suggestion
            print("Skipping", comp_label)
```

File: docker/setup_and_run_docker.sh [hunk 2]
```diff
@@ -50,6 +50,14 @@ if [ ! -f "parts.tar.gz" ]; then
...
 fi
 
+if [ ! -f "kit.tar.gz" ]; then
+  echo -e "\nPACKAGING KIT CODE..."
+  tar -cvzf kit.tar.gz \
+    runners \
+    -C "$ROOT" \
+    kit recipes default.config
+fi
+
```
Comment (hamishun) [inline docker/setup_and_run_docker.sh:60]: This can be brought in to parts.tar.gz

**Review by hamishun — CHANGES_REQUESTED** [review]

File: docker/setup_and_run_docker.sh (hunk 2 above)
Comment (denCalderon) [reply docker/setup_and_run_docker.sh]: Updated as it was requested

**Review by hamishun — APPROVED** [review]
LG

File: kit/hekit.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/python3
 
-# Copyright (C) 2020-2022 Intel Corporation
+# Copyright (C) 2020 Intel Corporation
```
Comment (jlhcrawford) [inline kit/hekit.py:3]:
```suggestion
# Copyright (C) 2021 Intel Corporation
```

File: recipes/default.toml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2020-2022 Intel Corporation
+# Copyright (C) 2020 Intel Corporation
```
Comment (jlhcrawford) [inline recipes/default.toml:1]:
```suggestion
# Copyright (C) 2022 Intel Corporation
```

File: recipes/test.toml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2020-2022 Intel Corporation
+# Copyright (C) 2020 Intel Corporation
```
Comment (jlhcrawford) [inline recipes/test.toml:1]:
```suggestion
# Copyright (C) 2022 Intel Corporation
```

**Review by jlhcrawford — CHANGES_REQUESTED** [review]
The rule for the copyright years is they should be from the year the file came into existence.

**Review by jlhcrawford — APPROVED** [review]

**Pull request merged**